plugins/css-blank-pseudo/src/browser.js

```diff
@@ -1,8 +1,14 @@
-/* global document,MutationObserver */
+/* global document,window,self,MutationObserver */
 import isValidReplacement from './is-valid-replacement.mjs';
 
 // form control elements selector
-const BLANK_CANDIDATES = 'input,select,textarea';
+function isFormControlElement(element) {
```

Comment:
trivial equivalent for `matches()` that isn't a polyfill and thus doesn't require testing or validation